homework/src/index.js

```diff
@@ -1,47 +1,110 @@
-'use strict';
+"use strick";
+
+function main(url) {
```

Comment:
You should not embed all functions inside one gigantic `main()` function. The skeleton code that was provided already had the correct structure.

Reply:
This is still not fixed.